linode_api4/groups/longview.py

```diff
@@ -60,3 +64,37 @@ def subscriptions(self, *filters):
         :rtype: PaginatedList of LongviewSubscription
         """
         return self.client._get_and_filter(LongviewSubscription, *filters)
+
+    def longview_plan_update(self, longview_subscription):
+        """
+        Update your Longview plan to that of the given subcription ID.
+
+        :param longview_subscription: The subscription ID for a particular Longview plan.
+                                      A value of null corresponds to Longview Free.
+        :type longview_subscription: str
```

Comment:
probably adding the `returns` and `rtype` as well?

Reply:
Fixed!